### ISSUE TITLE
Release 2.0.0-beta1

### ISSUE THREAD

**Issue by micbar** [owncloud/ocis#3627]
## PRs which need to be merged
<details>
  <summary>Click to expand</summary>

- [x] #3637
- [x] #3626 
- [x] #3551 
- [x] #3619
- [x] #3606 
- [x] #3602 
- [x] #3594 (nice to have)
- [x] https://github.com/cs3org/reva/pull/2798
- [x] #3635 @aduffeck @butonic 
- [x] https://github.com/cs3org/reva/pull/2799
- [x] #3629 -> merged as part of #3637
- [x] https://github.com/cs3org/reva/pull/2800
- [x] https://github.com/owncloud/ocis/pull/3643
- [x] https://github.com/owncloud/ocis/pull/3589
- [x] https://github.com/cs3org/reva/pull/2816
- [x] https://github.com/owncloud/ocis/pull/3648
- [x] #3656 
- [x] https://github.com/owncloud/ocis/pull/3672
- [x] https://github.com/owncloud/ocis/pull/3673

- [x] https://github.com/owncloud/ocis/pull/3678
- [x]  #2918 
- [x] https://github.com/owncloud/ocis/issues/3738

</details>
## Needs Clarification

- [x] Autoprovisioning of users when deployed with an external IdP #3540 
  -> will be fixed in beta2
## Release Process
- [x] Bump web to `v5.5.0-rc3` https://github.com/owncloud/ocis/pull/3680
- [x] Tag `v2.0.0-alpha1` from master branch
- [x] Brief marketing heads up 
- [x] Release Reva `v2.3.0` https://github.com/cs3org/reva/pull/2812
- [x] Update Reva to `v2.3.0` https://github.com/owncloud/ocis/pull/3681
   - [x] clean up our changelog regarding reva updates (there should be only one)
- [x] Update web to 5.5.0-rc4
- [x] Release Reva `v2.3.1`
- [x] Release Web `v5.5.0` https://github.com/owncloud/web/issues/6812
- [x] Update Reva  to `v2.3.1` https://github.com/owncloud/ocis/pull/3733
- [x] Update Web  to `v5.5.0-rc5`
- [x] Create branch `release-2.0.0-beta1` -> CODEFREEZE
- [x] Prepare Changelog 
- [x] Release Notes @pmaier1 https://github.com/owncloud/ocis/pull/3758
- [x] Create pre release tag `v2.0.0-alpha2`
- [x] Smoke Test @ScharfViktor 
  - [x] with oCIS WOPI server @wkloucek
  - [x] with oCIS Hello extension @wkloucek -> failed, oCIS hello needs love
- [x] Compatibility test / upgrade test @wkloucek -> failed, ocis fs disk layout has changed https://github.com/cs3org/reva/pull/2763
- [x] Update deployment examples https://github.com/owncloud/ocis/pull/3751
- [x] run K6 on the tag @fschade 
- [x] If needed, fix issues or write down known issues alert in chat
   - [x] https://github.com/owncloud/ocis/issues/3688 -> fix in https://github.com/owncloud/ocis/pull/3694
   - [x] fix deployment examples https://github.com/owncloud/ocis/pull/3690
   - [x] test parallel deployment and enable it in CI https://github.com/owncloud/ocis/pull/3666
- [x] Create final signed tag `v2.0.0-beta1`  
- [x] Merge release notes
  - [x] Check successful CI run on `v2.0.0-beta1` tag 
- [x] Ping marketing to update all download links  
- [x] Merge back release branch https://github.com/owncloud/ocis/pull/3750
- [x] Ping @hodyroff once the demo instances are running this release

**Comment by butonic** [comment]
Do we need the various owncloud drivers for migration / parallel deployment? Especially being able to read users from an owncloud database would require a database backend for libreidm.

**Comment by rtest12** [comment]
sso fixed?

**Comment by butonic** [comment]
@rtest12 sso in what regard? Do you have an issue link?

**Comment by rtest12** [comment]
> @rtest12 sso in what regard? Do you have an issue link?

#3540

**Comment by micbar** [comment]
@rtest12 this is currently a known issue. I will not guarantee that this is fixed in `2.0.0-beta1` but definately during the beta phase-

**Comment by wkloucek** [comment]
# Smoke test with WOPI server
- [ ] [Docker-Compose Setup](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_wopi) with https://github.com/owncloud/ocis/pull/3522 and https://github.com/cs3org/wopiserver/pull/74
- [ ] Login as Einstein
   - [ ] ~Go to all files -> Click "Create new file" -> You have multiple proposed office formats~ Not yet implemented
   - [ ] Upload an `.odt` file ~Create a new "odt" office file~
      - [ ] ~file opens~ open file in Collabora
      - [ ] add some text, save the file and close the editor / tab
      - [ ] reopen the newly created file
      - [ ] the previously added text is still there
   - [ ] ~Upload an `.md` file~ Create a new "md" file
      - [ ] ~file opens~ open file in CodiMD
      - [ ] add some text, save the file and close the editor / tab
      - [ ] reopen the newly created file
      - [ ] the previously added text is still there
   - [ ] Upload an `.docx` file ~Create a new "docx" file~
      - [ ] ~file opens~ open file in OnlyOffice
      - [ ] add some text, save the file and close the editor / tab
      - [ ] reopen the newly created file
      - [ ] the previously added text is still there
      - [ ] copy the URL of the open file view in OnlyOffice
      - [ ] open the URL in a new incognito tab (with no session)
      - [ ] be redirected to the login screen and then login in as Einstein 
      - [ ] see the file opened in the original view, where you copied the URL from
    - [ ] default actions
       - [ ] left-click on the `.odt` file
       - [ ] the `.odt` file is opened in Collabora
       - [ ] left-click on the `.docx` file
       - [ ] the `.docx` file is opened in OnlyOffice
       - [ ] left-click on the `.md` file 
       - [ ] the `.md` file is opened in CodiMD -> instead opened in the "SimpleEditor"
       - [ ] upload a `.jpg` file
       - [ ] left-click on the `.jpg` file
       - [ ] the `.jpg` file is opened in the MediaViewer
    - [ ] public links
       - [ ] Create a new folder "documents"
       - [ ] Upload ~Create~ a new "collaborative.odt" office file in the folder
       - [ ] Upload ~Create~ a new "collaborative.docx" office file in the folder
       - [ ] Upload ~Create~ a new "collaborative.md" file in the folder
       - [ ] share the folder "documents" with a public link (viewer role, no password, no expiration)
         - [ ] open the public link in an incognito tab
         - [ ] open the `.odt` file in Collabora
         - [ ] The `.odt` file is opened in Collabora in view only mode
         - [ ] open the `.docx` file in OnlyOffice
         - [ ] The `.docx` file is opened in OnlyOffice in view only mode
         - [ ] open the `.md` file in CodiMD
         - [ ] The `.md` file is opened in CodiMD in view only mode
         - [ ] copy the URL of the open file view in CodiMD
         - [ ] close the incognito window (clear session)
         - [ ] open the URL in a new incognito window
         - [ ] The `.md` file is opened in CodiMD in view only mode
       - [ ] share the folder "documents" with a public link (editor role, no password, no expiration)
         - [ ] open the public link in an incognito tab
         - [ ] open the `.odt` file in Collabora
         - [ ] The `.odt` file is opened in Collabora in edit mode and edits can be saved 
         - [ ] open the `.docx` file in OnlyOffice
         - [ ] The `.docx` file is opened in OnlyOffice in edit mode and edits can be saved
         - [ ] open the `.md` file in CodiMD
         - [ ] The `.md` file is opened in CodiMD in edit mode and edits can be saved
         - [ ] copy the URL of the open file view in CodiMD
         - [ ] close the incognito window (clear session)
         - [ ] open the URL in a new incognito window
         - [ ] The `.md` file is opened in CodiMD in edit mode and edits can be saved
       - [ ] share the folder "documents" with a public link (editor role, with password, no expiration)
         - [ ] open the public link in an incognito tab
         - [ ] enter public share password
         - [ ] open the `.odt` file in Collabora
         - [ ] The `.odt` file is opened in Collabora in edit mode
         - [ ] The `.odt` file is opened in Collabora in edit mode and edits can be saved 
         - [ ] open the `.md` file in CodiMD
         - [ ] The `.md` file is opened in CodiMD in edit mode
         - [ ] The `.md` file is opened in CodiMD in edit mode and edits can be saved
         - [ ] open the `.docx` file in OnlyOffice
         - [ ] The `.docx` file is opened in OnlyOffice in edit mode
         - [ ] The `.docx` file is opened in OnlyOffice in edit mode and edits can be saved 
         - [ ] copy the URL of the open file view in OnlyOffice
         - [ ] close the incognito window (clear session)
         - [ ] open the URL in a new incognito window
         - [ ] enter public share password
         - [ ] The `.docx` file is opened in OnlyOffice in edit mode and edits can be saved 
  - [ ] Create a new folder "documents"
    - [ ] Share the folder "documents" with Marie with editor permissions
    - [ ] Upload ~Create~ a new "collaborative.odt" office file in the folder
    - [ ] Upload ~Create~ a new "collaborative.docx" office file in the folder
    - [ ] Upload ~Create~ a new "collaborative.md" file in the folder
    - [ ] open "collaborative.odt" with Collabora and edit some text
    - [ ] open "collaborative.docx" with OnlyOffice and edit some text
    - [ ] open "collaborative.md" with CodiMD and edit some text
- [ ] Login as Marie in a different browser / incognito window
   - [ ] accept "documents" share from Einstein
   - [ ] navigate to "documents" share and open "collaborative.odt" with Collabora
      - [ ] see that Einstein is already editing it and see its cursor and name
      - [ ] changes will be shared between both users' editing sessions
   - [ ] navigate to "documents" share and open "collaborative.docx" with OnlyOffice
      - [ ] see that Einstein is already editing it and see its cursor and name
      - [ ] changes will be shared between both users' editing sessions
   - [ ] navigate to "documents" share and open "collaborative.md" with CodiMD
      - [ ] see that Einstein is already editing it and see its cursor and name
      - [ ] changes will be shared between both users' editing sessions

## Outcome:
- CodiMD does not support locking properly
- Read only public links open the editor in readwrite mode (all editors are affected)
- a bookmarked editor on publiclinks with password don't work

**Comment by wkloucek** [comment]
# Smoke test with [oCIS Hello](https://github.com/owncloud/ocis-hello)
- [ ] [Docker-Compose Setup](https://github.com/owncloud/ocis/tree/master/deployments/examples/ocis_hello)
- [ ] Login as Einstein
   - [ ] Navigate to Hello Extension (via app drawer)
   - [ ] Type "foo" and submit -> receive "Hello foo" (submit button does not work, hitting enter does work)
   - [ ] Navigate to settings and make sure there are no Hello configuration options
- [ ] Login as Admin
   - [ ] Navigate to Hello Extension (via app drawer)
   - [ ] Type "bar" and submit -> receive "Hello bar"
   - [ ] Navigate to settings
   - [ ] Navigate to Hello Extension (via app drawer)
   - [ ] Type "bar" and submit -> receive "Bonjour bar" (still Hello bar, the hello backend logs: `ReadValueByUniqueIdentifiers not implemented`)
- [ ] Login as Einstein
   - [ ] Navigate to Hello Extension (via app drawer)
   - [ ] Type "foo" and submit -> receive "Bonjour foo" (same as for admin)

## Findings
- Frontend design and layout is off, but functionality is ok (except the submit button). PR for design and layout already is WIP: https://github.com/owncloud/ocis-hello/pull/130
- Hello backends has following error: `ReadValueByUniqueIdentifiers not implemented`

**Comment by wkloucek** [comment]
# Compatibility test from v1.20.0 to v2.0.0-alpha1

## Create these with the original oCIS version and see if still the same after upgrading:
- shares
  - [ ] public share
  - [ ] share with user as role editor
  - [ ] share with user as role viewer
- settings
  - [ ] language settings
- files
  - [ ] uploaded folder & files
  - [ ] trashed files
- users
  - [ ] non demo user with role admin
  - [ ] non demo user with role user

## additional issues:
-

**Comment by micbar** [comment]
# Changelog Draft

## ocis

* Bugfix - Idp: Check if CA certificate if present: [#3623](https://github.com/owncloud/ocis/issues/3623)
* Bugfix - Return proper errors when ocs/cloud/users is using the cs3 backend: [#3483](https://github.com/owncloud/ocis/issues/3483)
* Bugfix - URL encode the webdav url in the graph API: [#3597](https://github.com/owncloud/ocis/pull/3597)
* Change - Update ocis packages and imports to V2: [#3678](https://github.com/owncloud/ocis/pull/3678)
* Change - Load configuration files just from one directory: [#3587](https://github.com/owncloud/ocis/pull/3587)
* Change - Reduce permissions on docker image predeclared volumes: [#3641](https://github.com/owncloud/ocis/pull/3641)
* Change - Introduce `ocis init` and remove all default secrets: [#3551](https://github.com/owncloud/ocis/pull/3551)
* Change - Reduce drives in graph /me/drives API: [#3629](https://github.com/owncloud/ocis/pull/3629)
* Change - Switched default configuration to use libregraph/idm: [#3331](https://github.com/owncloud/ocis/pull/3331)
* Change - Rename MetadataUserID: [#3671](https://github.com/owncloud/ocis/pull/3671)
* Change - Use new space ID util functions: [#3648](https://github.com/owncloud/ocis/pull/3648)
* Change - Rename serviceUser to systemUser: [#3673](https://github.com/owncloud/ocis/pull/3673)
* Change - Split MachineAuth from SystemUser: [#3672](https://github.com/owncloud/ocis/pull/3672)
* Enhancement - Align service naming: [#3606](https://github.com/owncloud/ocis/pull/3606)
* Enhancement - Wrap metadata storage with dedicated reva gateway: [#3602](https://github.com/owncloud/ocis/pull/3602)
* Enhancement - Add initial version of the search extensions: [#3635](https://github.com/owncloud/ocis/pull/3635)
* Enhancement - Add capability for public link single file edit: [#6787](https://github.com/owncloud/web/pull/6787)
* Enhancement - Added `share_jail` and `projects` feature flags in spaces capability: [#3626](https://github.com/owncloud/ocis/pull/3626)
* Enhancement - Update linkshare capabilities: [#3579](https://github.com/owncloud/ocis/pull/3579)
* Enhancement - Update reva to v2.3.0: [#3552](https://github.com/owncloud/ocis/pull/3552)
* Enhancement - Update ownCloud Web to v5.5.0-rc.3: [#6854](https://github.com/owncloud/web/pull/6854)

## reva

* Bugfix [cs3org/reva#2693](https://github.com/cs3org/reva/pull/2693): Support editnew actions from MS Office
* Bugfix [cs3org/reva#2588](https://github.com/cs3org/reva/pull/2588): Dockerfile.revad-ceph to use the right base image
* Bugfix [cs3org/reva#2499](https://github.com/cs3org/reva/pull/2499): Removed check DenyGrant in resource permission
* Bugfix [cs3org/reva#2285](https://github.com/cs3org/reva/pull/2285): Accept new userid idp format
* Bugfix [cs3org/reva#2802](https://github.com/cs3org/reva/pull/2802): Bugfix the resource id handling for space shares
* Bugfix [cs3org/reva#2800](https://github.com/cs3org/reva/pull/2800): Bugfix spaceid parsing in spaces trashbin API
* Bugfix [cs3org/reva#2608](https://github.com/cs3org/reva/pull/2608): Respect the tracing_service_name config variable
* Bugfix [cs3org/reva#2742](https://github.com/cs3org/reva/pull/2742): Use exact match in login filter
* Bugfix [cs3org/reva#2759](https://github.com/cs3org/reva/pull/2759): Made uid, gid claims parsing more robust in OIDC auth provider
* Bugfix [cs3org/reva#2788](https://github.com/cs3org/reva/pull/2788): Return the correct file IDs on public link resources
* Bugfix [cs3org/reva#2322](https://github.com/cs3org/reva/pull/2322): Use RFC3339 for parsing dates
* Bugfix [cs3org/reva#2784](https://github.com/cs3org/reva/pull/2784): Disable storageprovider cache for the share jail
* Bugfix [cs3org/reva#2555](https://github.com/cs3org/reva/pull/2555): Bugfix site accounts endpoints
* Bugfix [cs3org/reva#2675](https://github.com/cs3org/reva/pull/2675): Updates Makefile according to latest go standards
* Bugfix [cs3org/reva#2572](https://github.com/cs3org/reva/pull/2572): Wait for nats server on middleware start
* Change [cs3org/reva#2735](https://github.com/cs3org/reva/pull/2735): Avoid user enumeration
* Change [cs3org/reva#2737](https://github.com/cs3org/reva/pull/2737): Bump go-cs3api
* Change [cs3org/reva#2763](https://github.com/cs3org/reva/pull/2763): Change the oCIS and S3NG  storage driver blob store layout
* Change [cs3org/reva#2596](https://github.com/cs3org/reva/pull/2596): Remove hash from public link urls
* Change [cs3org/reva#2785](https://github.com/cs3org/reva/pull/2785): Implement workaround for chi.RegisterMethod
* Change [cs3org/reva#2559](https://github.com/cs3org/reva/pull/2559): Do not encode webDAV ids to base64
* Change [cs3org/reva#2740](https://github.com/cs3org/reva/pull/2740): Rename oc10 share manager driver
* Change [cs3org/reva#2561](https://github.com/cs3org/reva/pull/2561): Merge oidcmapping auth manager into oidc
* Enhancement [cs3org/reva#2698](https://github.com/cs3org/reva/pull/2698): Make capabilities endpoint public, authenticate users is present
* Enhancement [cs3org/reva#2515](https://github.com/cs3org/reva/pull/2515): Enabling tracing by default if not explicitly disabled
* Enhancement [cs3org/reva#2686](https://github.com/cs3org/reva/pull/2686): Features for favorites xattrs in EOS, cache for scope expansion
* Enhancement [cs3org/reva#2494](https://github.com/cs3org/reva/pull/2494): Use sys ACLs for file permissions
* Enhancement [cs3org/reva#2522](https://github.com/cs3org/reva/pull/2522): Introduce events
* Enhancement [cs3org/reva#2811](https://github.com/cs3org/reva/pull/2811): Add event for created directories
* Enhancement [cs3org/reva#2798](https://github.com/cs3org/reva/pull/2798): Add additional fields to events to enable search
* Enhancement [cs3org/reva#2790](https://github.com/cs3org/reva/pull/2790): Fake providerids so API stays stable after beta
* Enhancement [cs3org/reva#2685](https://github.com/cs3org/reva/pull/2685): Enable federated account access
* Enhancement [cs3org/reva#1787](https://github.com/cs3org/reva/pull/1787): Add support for HTTP TPC
* Enhancement [cs3org/reva#2799](https://github.com/cs3org/reva/pull/2799): Add flag to enable unrestriced listing of spaces
* Enhancement [cs3org/reva#2560](https://github.com/cs3org/reva/pull/2560): Mentix PromSD extensions
* Enhancement [cs3org/reva#2741](https://github.com/cs3org/reva/pull/2741): Meta path for user
* Enhancement [cs3org/reva#2613](https://github.com/cs3org/reva/pull/2613): Externalize custom mime types configuration for storage providers
* Enhancement [cs3org/reva#2163](https://github.com/cs3org/reva/pull/2163): Nextcloud-based share manager for pkg/ocm/share
* Enhancement [cs3org/reva#2696](https://github.com/cs3org/reva/pull/2696): Preferences driver refactor and cbox sql implementation
* Enhancement [cs3org/reva#2052](https://github.com/cs3org/reva/pull/2052): New CS3API datatx methods
* Enhancement [cs3org/reva#2743](https://github.com/cs3org/reva/pull/2743): Add capability for public link single file edit
* Enhancement [cs3org/reva#2738](https://github.com/cs3org/reva/pull/2738): Site accounts site-global settings
* Enhancement [cs3org/reva#2672](https://github.com/cs3org/reva/pull/2672): Further Site Accounts improvements
* Enhancement [cs3org/reva#2549](https://github.com/cs3org/reva/pull/2549): Site accounts improvements
* Enhancement [cs3org/reva#2795](https://github.com/cs3org/reva/pull/2795): Add feature flags "projects" and "share_jail" to spaces capability
* Enhancement [cs3org/reva#2514](https://github.com/cs3org/reva/pull/2514): Reuse ocs role objects in other drivers
* Enhancement [cs3org/reva#2781](https://github.com/cs3org/reva/pull/2781): In memory user provider
* Enhancement [cs3org/reva#2752](https://github.com/cs3org/reva/pull/2752): Refactor the rest user and group provider drivers

## web

* Bugfix - Apply text selection range for new files: [#6756](https://github.com/owncloud/web/issues/6756)
* Bugfix - Decrease build time and fix faulty oc10 docker dev env: [#6855](https://github.com/owncloud/web/pull/6855)
* Bugfix - Do not load files from cache: [#6447](https://github.com/owncloud/web/pull/6447)
* Bugfix - Indicate guest shares: [#6813](https://github.com/owncloud/web/pull/6813)
* Bugfix - Edit public links with expiration: [#6858](https://github.com/owncloud/web/issues/6858)
* Bugfix - Password enforcement for public links: [#6323](https://github.com/owncloud/web/issues/6323)
* Bugfix - Rename is clickable on mobile: [#6767](https://github.com/owncloud/web/issues/6767)
* Bugfix - Rename a file in favorites list with same name but in different folder: [#6804](https://github.com/owncloud/web/pull/6804)
* Bugfix - Resetting store on logout: [#6694](https://github.com/owncloud/web/pull/6694)
* Bugfix - Space image not shown if file name contains blanks: [#6881](https://github.com/owncloud/web/pull/6881)
* Bugfix - Spaces Contextmenu trigger id isn't valid: [#6845](https://github.com/owncloud/web/issues/6845)
* Bugfix - Upload overlay links: [#6846](https://github.com/owncloud/web/pull/6846)
* Enhancement - Add OcContextualHelper: [#6590](https://github.com/owncloud/web/issues/6590)
* Enhancement - Add show file extension toggle switch in file list settings: [#6793](https://github.com/owncloud/web/pull/6793)
* Enhancement - Add un-share confirmation dialog: [#6795](https://github.com/owncloud/web/pull/6795)
* Enhancement - Capability-based searchbar rendering: [#6856](https://github.com/owncloud/web/pull/6856)
* Enhancement - Design polishing: [#6781](https://github.com/owncloud/web/pull/6781)
* Enhancement - Enable search all files for ocis backend: [#6841](https://github.com/owncloud/web/pull/6841)
* Enhancement - Fixed aspect ratio for spaces images: [#6829](https://github.com/owncloud/web/pull/6829)
* Enhancement - Introduce quicklinks: [#6820](https://github.com/owncloud/web/pull/6820)
* Enhancement - Introduce user-management app: [#6673](https://github.com/owncloud/web/issues/6673)
* Enhancement - Make rename resource icon always visible instead just on hover: [#6817](https://github.com/owncloud/web/pull/6817)
* Enhancement - Polish the upload overlay: [#6837](https://github.com/owncloud/web/pull/6837)
* Enhancement - Redesign link sharing: [#6749](https://github.com/owncloud/web/pull/6749)
* Enhancement - Refactor upload input components: [#6859](https://github.com/owncloud/web/pull/6859)
* Enhancement - Get rid of the integration tests: [#6863](https://github.com/owncloud/web/pull/6863)
* Enhancement - Remove the upload progress component: [#6825](https://github.com/owncloud/web/pull/6825)
* Enhancement - Replace deprecated String.prototype.substr(): [#6718](https://github.com/owncloud/web/pull/6718)
* Enhancement - Resumable uploads: [#6202](https://github.com/owncloud/web/pull/6202)
* Enhancement - Space store improvements: [#6868](https://github.com/owncloud/web/pull/6868)
* Enhancement - Update ODS to v13.1.0-rc.5: [#6749](https://github.com/owncloud/web/pull/6749)
* Enhancement - Update SDK: [#6820](https://github.com/owncloud/web/pull/6820)
* Enhancement - Use event bus for upload related actions: [#6853](https://github.com/owncloud/web/pull/6853)

**Comment by micbar** [comment]
# 🎉  2.0.0-alpha2 available

## Github

https://github.com/owncloud/ocis/releases/tag/v2.0.0-alpha2

## Mirrors 

https://download.owncloud.com/ocis/ocis/testing/2.0.0-alpha2/

## Docker

```
docker pull owncloud/ocis:2.0.0-alpha2
```

**Comment by kobergj** [comment]
Smoke testing the audit service brought up some issues. None of them is critical in my opinion, but should be revisited. 
https://github.com/owncloud/ocis/issues/3753

**Comment by rhafer** [comment]
I briefly tested the basic search functionality with `owncloud/ocis:2.0.0-alpha2`, found a couple (already known) issues (e.g. #3700, #3701). But the basic functionality is there.

**Comment by ScharfViktor** [comment]
Smoke
branch: v2.0.0-alpha2

### Case 1: admin share folder(with file) to marie:

Admin:
- [x] upload big file 10gb - slow and uploads partly because of page refreshing
-🚫 [ ] upload folder with 803 small images. - sometimes POST get `412 Precondition Failed`
- [x] Check pagination, count items per page
- [x] upload folder(drag folder) "folder_to_share" with file.pdf
- [x] share folder "folder_to_share" to marie with edit right

Marie:
- [x] accept "folder_to_share"
- [x] rename "folder_to_share" - can rename only inside of the folder
- [x] upload new file "file.txt" to "folder_to_share" (admin see new file)
- [x] open pdf file
- [x] rename file.pdf (use rigth click) (admin see that file changed name)

Admin:
- [x] add new file.jpeg to "folder_to_share"
- [x] move file.pdf to root "personal" (marie can not see file.pdf) - doesn't work
- [x]  add new "file.txt" create new version. (use drag and drop)
- [x] open txt file in markdown-editor. Change and save.

Marie:
- [ ] 🚫 can download both versions (!error 501)
- [x] restore old version

Admin:
- [x] change share "folder_to_share" from "edit" to "viewer" (marie cannot rename/move/delete/download file in folder and cannot create new document in the folder)

Marie:
- [x] decline folder (removed from shares folder)
- [x] accept again 

Admin:
- [x] delete folder "folder_to_share" (removed from shared with me, removed from Marie)

## Case 2: admin share file to marie:

Admin:
- [x] upload file.jpeg
- [x] share file.jpeg to marie with rigth viewer (use main menu. find marie by email marie@example.org)

Marie:
- [x] accept file (marie can open/download/copy file but cannot delete/move/rename)
- [x] copy file to "/"
- [x] open file in Mediaviewer
- [x] download file

Admin:
- [x] share share "file.txt" with edit right

Marie:
- [x] rename file (🚫admin don't see)
- [x] change content of the file (can see new version)

Admin:
- [x] unshare "file.jpeg" (marie cannot see "file.jpeg" in Shares/Shared for me but see the copied file)

## Case 3: admin create public links to folder

Admin:
- [x] create folder "for upload"
- [x] add link with name "to_upload" with role "Uploader" with pass and expiration date
- [x] open link (can only upload)
- [x] upload file (admin can see new file)
- [x] refresh page(page is empty)
- [x] change share "for upload" from "Uploader" to "edit"
- [x] admin add new file to "for upload"
- [x] refresh link (file exist)
- [x] admin change pass (refresh link has to type new pass)

## Case 4: admin create link to file

Admin:
- [x] create file png
- [x] add link with main menu
- [x] open link(can open, download, see) in the incognito tab
- [x] change link, add pass
- [x] refresh page (input pass)
- [x] delete link
- [x] refresh page (resource not found)
- [x] delete file
- [x] restore
- [x] delete
- [x] Empty trash

## Case 5: space story:
Katherine:
- [x] create space
- [x] add space image
- [x] add subtitle "for test"
- [x] edit description
- [x] try to upload file (more 1 Gb) - (507- Insufficient Storage)
- [x] increase quota to 5Gb and try again
- [x] delete file
- [x] add member Marie with edit role
- [x]  add member Einstein with viewer role

Marie:
- [x] can see space without accepting
- [x] change image and description
- [x] can upload file/folder and create file and folder
- [x] cannot create resource sharing and space sharing
- [x] can see deleted files and can restore files from trash
- [x] cannot disable space

Einstein:
- [x] can see space without accepting
- [x] cannot change image and description
- [x] cannot upload file/folder and create file and folder
- [x] cannot create resource sharing and space sharing
- [x] can see deleted files and cannot restore files from trash
- [x] cannot disable space

Admin change Einstein role to manager:

Einstein:
- [x] can see space without accepting
- [x] can change image and description
- [x] can upload file/folder and create file and folder
- [x] can create resource sharing and space sharing
- [x] can see deleted files and can restore files from trash
- [x] can change admins role and delete share
- [x] can disable and delete space 


## Case 6: create new user via accounts:
Admin (test via api):
- [x]  create new account "tom" use graph Api
- [x] create new group
- [x] add user to group
- [x] share folder with group
- [x] change language
- [x] delete users


it's all good. without blocks

**Comment by micbar** [comment]
# Known issues 📜 

- COPY&MOVE is broken outside of personal spaces
- Single File Shares have no download action and thumbnail is broken for the receiver
- We cannot rename shares -> Regression, needs concept, not possible with WEBDAV
- The default share manager still stores the data in json, the cs3 share manager is available but we need to enable it by default. Needs a migration of shares between beta releases.

**Comment by pmaier1** [comment]
> Single File Shares have no download action and thumbnail is broken for the receiver

What is a single file share? Public link or user? Ticket?

**Comment by micbar** [comment]
# Infinite Scale `v2.0.0-beta1` is released 🎉 

## GitHub Release

- https://github.com/owncloud/ocis/releases/tag/v2.0.0-beta1

## Docker

```sh
docker pull owncloud/ocis:2.0.0-beta1
```

## Download Mirrors

https://download.owncloud.com/ocis/ocis/testing/2.0.0-beta1


# Changelog for [2.0.0-beta1] (2022-05-10)

The following sections list the changes for 2.0.0-beta1.

[2.0.0-beta1]: https://github.com/owncloud/ocis/compare/v1.20.0...v2.0.0-beta1

## Summary

* Bugfix - Remove runtime kill and run commands: [#3740](https://github.com/owncloud/ocis/pull/3740)
* Bugfix - Check permissions when deleting Space: [#3709](https://github.com/owncloud/ocis/pull/3709)
* Bugfix - Make IDP secrets configurable via environment variables: [#3744](https://github.com/owncloud/ocis/pull/3744)
* Bugfix - Thumbnails for `/dav/xxx?preview=1` requests: [#3567](https://github.com/owncloud/ocis/pull/3567)
* Bugfix - Idp: Check if CA certificate if present: [#3623](https://github.com/owncloud/ocis/issues/3623)
* Bugfix - Return proper errors when ocs/cloud/users is using the cs3 backend: [#3483](https://github.com/owncloud/ocis/issues/3483)
* Bugfix - URL encode the webdav url in the graph API: [#3597](https://github.com/owncloud/ocis/pull/3597)
* Change - Update ocis packages and imports to V2: [#3678](https://github.com/owncloud/ocis/pull/3678)
* Change - Load configuration files just from one directory: [#3587](https://github.com/owncloud/ocis/pull/3587)
* Change - Reduce permissions on docker image predeclared volumes: [#3641](https://github.com/owncloud/ocis/pull/3641)
* Change - Introduce `ocis init` and remove all default secrets: [#3551](https://github.com/owncloud/ocis/pull/3551)
* Change - Reduce drives in graph /me/drives API: [#3629](https://github.com/owncloud/ocis/pull/3629)
* Change - Switched default configuration to use libregraph/idm: [#3331](https://github.com/owncloud/ocis/pull/3331)
* Change - Rename MetadataUserID: [#3671](https://github.com/owncloud/ocis/pull/3671)
* Change - Use new space ID util functions: [#3648](https://github.com/owncloud/ocis/pull/3648)
* Change - Rename serviceUser to systemUser: [#3673](https://github.com/owncloud/ocis/pull/3673)
* Change - Split MachineAuth from SystemUser: [#3672](https://github.com/owncloud/ocis/pull/3672)
* Enhancement - Align service naming: [#3606](https://github.com/owncloud/ocis/pull/3606)
* Enhancement - Wrap metadata storage with dedicated reva gateway: [#3602](https://github.com/owncloud/ocis/pull/3602)
* Enhancement - Product field in OCS version: [#2918](https://github.com/owncloud/ocis/pull/2918)
* Enhancement - Add initial version of the search extensions: [#3635](https://github.com/owncloud/ocis/pull/3635)
* Enhancement - Add capability for public link single file edit: [#6787](https://github.com/owncloud/web/pull/6787)
* Enhancement - Added `share_jail` and `projects` feature flags in spaces capability: [#3626](https://github.com/owncloud/ocis/pull/3626)
* Enhancement - Update linkshare capabilities: [#3579](https://github.com/owncloud/ocis/pull/3579)
* Enhancement - Update reva to v2.3.1: [#3552](https://github.com/owncloud/ocis/pull/3552)
* Enhancement - Update ownCloud Web to v5.5.0-rc.5: [#6854](https://github.com/owncloud/web/pull/6854)

## Details

* Bugfix - Remove runtime kill and run commands: [#3740](https://github.com/owncloud/ocis/pull/3740)

   We've removed the kill and run commands from the oCIS runtime. If these dynamic capabilities
   are needed, one should switch to a full fledged supervisor and start oCIS as individual
   services.

   If one wants to start a only a subset of services, this is still possible by setting
   OCIS_RUN_EXTENSIONS.

   https://github.com/owncloud/ocis/pull/3740

* Bugfix - Check permissions when deleting Space: [#3709](https://github.com/owncloud/ocis/pull/3709)

   Check for manager permissions when deleting spaces. Do not allow deleting spaces via dav
   service

   https://github.com/owncloud/ocis/pull/3709

* Bugfix - Make IDP secrets configurable via environment variables: [#3744](https://github.com/owncloud/ocis/pull/3744)

   We've fixed the configuration options of the IDP to make the IDP secrets again configurable via
   environment variables.

   https://github.com/owncloud/ocis/pull/3744

* Bugfix - Thumbnails for `/dav/xxx?preview=1` requests: [#3567](https://github.com/owncloud/ocis/pull/3567)

   We've added the thumbnail rendering for `/dav/xxx?preview=1`,
   `/remote.php/webdav/{relative path}?preview=1` and `/webdav/{relative
   path}?preview=1` requests, which was previously not supported because of missing routes. It
   now returns the same thumbnails as for `/remote.php/dav/xxx?preview=1`.

   https://github.com/owncloud/ocis/pull/3567

* Bugfix - Idp: Check if CA certificate if present: [#3623](https://github.com/owncloud/ocis/issues/3623)

   Upon first start with the default configurtation the idm service creates a server
   certificate, that might not be finished before the idp service is starting. Add a check to idp
   similar to what the user, group, and auth-providers implement.

   https://github.com/owncloud/ocis/issues/3623

* Bugfix - Return proper errors when ocs/cloud/users is using the cs3 backend: [#3483](https://github.com/owncloud/ocis/issues/3483)

   The ocs API was just exiting with a fatal error on any update request, when configured for the cs3
   backend. Now it returns a proper error.

   https://github.com/owncloud/ocis/issues/3483

* Bugfix - URL encode the webdav url in the graph API: [#3597](https://github.com/owncloud/ocis/pull/3597)

   Fixed the webdav URL in the drives responses. Without encoding the URL could be broken by files
   with spaces in the file name.

   https://github.com/owncloud/ocis/issues/3538
   https://github.com/owncloud/ocis/pull/3597

* Change - Update ocis packages and imports to V2: [#3678](https://github.com/owncloud/ocis/pull/3678)

   This needs to be done in preparation for the major version bump in ocis.

   https://github.com/owncloud/ocis/pull/3678

* Change - Load configuration files just from one directory: [#3587](https://github.com/owncloud/ocis/pull/3587)

   We've changed the configuration file loading behavior and are now only loading configuration
   files from ONE single directory. This directory can be set on compile time or via an environment
   variable on startup (`OCIS_CONFIG_DIR`).

   We are using following configuration default paths:

   - Docker images: `/etc/ocis/` - Binary releases: `$HOME/.ocis/config/`

   https://github.com/owncloud/ocis/pull/3587

* Change - Reduce permissions on docker image predeclared volumes: [#3641](https://github.com/owncloud/ocis/pull/3641)

   We've lowered the permissions on the predeclared volumes of the oCIS docker image from 777 to
   750.

   This change doesn't affect you, unless you use the docker image with the non default uid/guid to
   start oCIS (default is 1000:1000).

   https://github.com/owncloud/ocis/pull/3641

* Change - Introduce `ocis init` and remove all default secrets: [#3551](https://github.com/owncloud/ocis/pull/3551)

   We've removed all default secrets and the hardcoded UUID of the user `admin`. This means you
   can't start oCIS any longer without setting these via environment variable or configuration
   file.

   In order to make this easy for you, we introduced a new command: `ocis init`. You can run this
   command before starting oCIS with `ocis server` and it will bootstrap you a configuration file
   for a secure oCIS instance.

   https://github.com/owncloud/ocis/issues/3524
   https://github.com/owncloud/ocis/pull/3551
   https://github.com/owncloud/ocis/pull/3743

* Change - Reduce drives in graph /me/drives API: [#3629](https://github.com/owncloud/ocis/pull/3629)

   Reduced the drives in the graph `/me/drives` API to only the drives the user has access to. The
   endpoint `/drives` will list all drives when the user has the permission.

   https://github.com/owncloud/ocis/pull/3629

* Change - Switched default configuration to use libregraph/idm: [#3331](https://github.com/owncloud/ocis/pull/3331)

   We switched the default configuration of oCIS to use the "idm" service (based on
   libregraph/idm) as the standard source for user and group information. The accounts and
   glauth services are no longer enabled by default and will be removed with an upcoming release.

   https://github.com/owncloud/ocis/pull/3331
   https://github.com/owncloud/ocis/pull/3633

* Change - Rename MetadataUserID: [#3671](https://github.com/owncloud/ocis/pull/3671)

   MetadataUserID is renamed to SystemUserID including yaml tags and env vars

   https://github.com/owncloud/ocis/pull/3671

* Change - Use new space ID util functions: [#3648](https://github.com/owncloud/ocis/pull/3648)

   Changed code to use the new space ID util functions so that everything works with the new spaces
   ID format.

   https://github.com/owncloud/ocis/pull/3648
   https://github.com/owncloud/ocis/pull/3669

* Change - Rename serviceUser to systemUser: [#3673](https://github.com/owncloud/ocis/pull/3673)

   We renamed serviceUser to systemUser in all configs and vars including yaml-tags and env vars

   https://github.com/owncloud/ocis/pull/3673

* Change - Split MachineAuth from SystemUser: [#3672](https://github.com/owncloud/ocis/pull/3672)

   We now have two different APIKeys: MachineAuth for the machine-auth service and SystemUser
   for the system user used e.g. by settings service

   https://github.com/owncloud/ocis/pull/3672

* Enhancement - Align service naming: [#3606](https://github.com/owncloud/ocis/pull/3606)

   We now reflect the configured service names when listing them in the ocis runtime

   https://github.com/owncloud/ocis/issues/3603
   https://github.com/owncloud/ocis/pull/3606

* Enhancement - Wrap metadata storage with dedicated reva gateway: [#3602](https://github.com/owncloud/ocis/pull/3602)

   We wrapped the metadata storage in a minimal reva instance with a dedicated gateway, including
   static storage registry, static auth registry, in memory userprovider, machine
   authprovider and demo permissions service. This allows us to preconfigure the service user
   for the ocis settings service, share and public share providers.

   https://github.com/owncloud/ocis/pull/3602
   https://github.com/owncloud/ocis/pull/3647

* Enhancement - Product field in OCS version: [#2918](https://github.com/owncloud/ocis/pull/2918)

   We've added a new field to the OCS Version, which is supposed to announce the product name. The
   web ui as a client will make use of it to make the backend product and version available (e.g. for
   easier bug reports).

   https://github.com/owncloud/ocis/pull/2918

* Enhancement - Add initial version of the search extensions: [#3635](https://github.com/owncloud/ocis/pull/3635)

   It is now possible to search for files and directories by their name using the web UI. Therefor
   new search extension indexes files in a persistent local index.

   https://github.com/owncloud/ocis/pull/3635

* Enhancement - Add capability for public link single file edit: [#6787](https://github.com/owncloud/web/pull/6787)

   It is now possible to share a single file by link with edit permissions. Therefore we need a
   public share capability to enable that feature in the clients. At the same time, we improved the
   WebDAV permissions for public links.

   https://github.com/owncloud/web/pull/6787
   https://github.com/owncloud/ocis/pull/3538

* Enhancement - Added `share_jail` and `projects` feature flags in spaces capability: [#3626](https://github.com/owncloud/ocis/pull/3626)

   We've added feature flags to the `spaces` capability to indicate to clients which features are
   supposed to be shown to users.

   https://github.com/owncloud/ocis/pull/3626

* Enhancement - Update linkshare capabilities: [#3579](https://github.com/owncloud/ocis/pull/3579)

   We have updated the capabilities regarding password enforcement and expiration dates of
   public links. They were previously hardcoded in a way that didn't reflect the actual backend
   functionality anymore.

   https://github.com/owncloud/ocis/pull/3579

* Enhancement - Update reva to v2.3.1: [#3552](https://github.com/owncloud/ocis/pull/3552)

   Updated reva to version 2.3.1. This update includes

  * Bugfix [cs3org/reva#2827](https://github.com/cs3org/reva/pull/2827): Check permissions when deleting spaces
  * Bugfix [cs3org/reva#2830](https://github.com/cs3org/reva/pull/2830): Correctly render response when accepting merged shares
  * Bugfix [cs3org/reva#2831](https://github.com/cs3org/reva/pull/2831): Fix uploads to owncloudsql storage when no mtime is provided
  * Enhancement [cs3org/reva#2833](https://github.com/cs3org/reva/pull/2833): Make status.php values configurable
  * Enhancement [cs3org/reva#2832](https://github.com/cs3org/reva/pull/2832): Add version option for ocdav go-micro service

   Updated reva to version 2.3.0. This update includes:

  * Bugfix [cs3org/reva#2693](https://github.com/cs3org/reva/pull/2693): Support editnew actions from MS Office
  * Bugfix [cs3org/reva#2588](https://github.com/cs3org/reva/pull/2588): Dockerfile.revad-ceph to use the right base image
  * Bugfix [cs3org/reva#2499](https://github.com/cs3org/reva/pull/2499): Removed check DenyGrant in resource permission
  * Bugfix [cs3org/reva#2285](https://github.com/cs3org/reva/pull/2285): Accept new userid idp format
  * Bugfix [cs3org/reva#2802](https://github.com/cs3org/reva/pull/2802): Bugfix the resource id handling for space shares
  * Bugfix [cs3org/reva#2800](https://github.com/cs3org/reva/pull/2800): Bugfix spaceid parsing in spaces trashbin API
  * Bugfix [cs3org/reva#2608](https://github.com/cs3org/reva/pull/2608): Respect the tracing_service_name config variable
  * Bugfix [cs3org/reva#2742](https://github.com/cs3org/reva/pull/2742): Use exact match in login filter
  * Bugfix [cs3org/reva#2759](https://github.com/cs3org/reva/pull/2759): Made uid, gid claims parsing more robust in OIDC auth provider
  * Bugfix [cs3org/reva#2788](https://github.com/cs3org/reva/pull/2788): Return the correct file IDs on public link resources
  * Bugfix [cs3org/reva#2322](https://github.com/cs3org/reva/pull/2322): Use RFC3339 for parsing dates
  * Bugfix [cs3org/reva#2784](https://github.com/cs3org/reva/pull/2784): Disable storageprovider cache for the share jail
  * Bugfix [cs3org/reva#2555](https://github.com/cs3org/reva/pull/2555): Bugfix site accounts endpoints
  * Bugfix [cs3org/reva#2675](https://github.com/cs3org/reva/pull/2675): Updates Makefile according to latest go standards
  * Bugfix [cs3org/reva#2572](https://github.com/cs3org/reva/pull/2572): Wait for nats server on middleware start
  * Change [cs3org/reva#2735](https://github.com/cs3org/reva/pull/2735): Avoid user enumeration
  * Change [cs3org/reva#2737](https://github.com/cs3org/reva/pull/2737): Bump go-cs3api
  * Change [cs3org/reva#2763](https://github.com/cs3org/reva/pull/2763): Change the oCIS and S3NG  storage driver blob store layout
  * Change [cs3org/reva#2596](https://github.com/cs3org/reva/pull/2596): Remove hash from public link urls
  * Change [cs3org/reva#2785](https://github.com/cs3org/reva/pull/2785): Implement workaround for chi.RegisterMethod
  * Change [cs3org/reva#2559](https://github.com/cs3org/reva/pull/2559): Do not encode webDAV ids to base64
  * Change [cs3org/reva#2740](https://github.com/cs3org/reva/pull/2740): Rename oc10 share manager driver
  * Change [cs3org/reva#2561](https://github.com/cs3org/reva/pull/2561): Merge oidcmapping auth manager into oidc
  * Enhancement [cs3org/reva#2698](https://github.com/cs3org/reva/pull/2698): Make capabilities endpoint public, authenticate users is present
  * Enhancement [cs3org/reva#2515](https://github.com/cs3org/reva/pull/2515): Enabling tracing by default if not explicitly disabled
  * Enhancement [cs3org/reva#2686](https://github.com/cs3org/reva/pull/2686): Features for favorites xattrs in EOS, cache for scope expansion
  * Enhancement [cs3org/reva#2494](https://github.com/cs3org/reva/pull/2494): Use sys ACLs for file permissions
  * Enhancement [cs3org/reva#2522](https://github.com/cs3org/reva/pull/2522): Introduce events
  * Enhancement [cs3org/reva#2811](https://github.com/cs3org/reva/pull/2811): Add event for created directories
  * Enhancement [cs3org/reva#2798](https://github.com/cs3org/reva/pull/2798): Add additional fields to events to enable search
  * Enhancement [cs3org/reva#2790](https://github.com/cs3org/reva/pull/2790): Fake providerids so API stays stable after beta
  * Enhancement [cs3org/reva#2685](https://github.com/cs3org/reva/pull/2685): Enable federated account access
  * Enhancement [cs3org/reva#1787](https://github.com/cs3org/reva/pull/1787): Add support for HTTP TPC
  * Enhancement [cs3org/reva#2799](https://github.com/cs3org/reva/pull/2799): Add flag to enable unrestriced listing of spaces
  * Enhancement [cs3org/reva#2560](https://github.com/cs3org/reva/pull/2560): Mentix PromSD extensions
  * Enhancement [cs3org/reva#2741](https://github.com/cs3org/reva/pull/2741): Meta path for user
  * Enhancement [cs3org/reva#2613](https://github.com/cs3org/reva/pull/2613): Externalize custom mime types configuration for storage providers
  * Enhancement [cs3org/reva#2163](https://github.com/cs3org/reva/pull/2163): Nextcloud-based share manager for pkg/ocm/share
  * Enhancement [cs3org/reva#2696](https://github.com/cs3org/reva/pull/2696): Preferences driver refactor and cbox sql implementation
  * Enhancement [cs3org/reva#2052](https://github.com/cs3org/reva/pull/2052): New CS3API datatx methods
  * Enhancement [cs3org/reva#2743](https://github.com/cs3org/reva/pull/2743): Add capability for public link single file edit
  * Enhancement [cs3org/reva#2738](https://github.com/cs3org/reva/pull/2738): Site accounts site-global settings
  * Enhancement [cs3org/reva#2672](https://github.com/cs3org/reva/pull/2672): Further Site Accounts improvements
  * Enhancement [cs3org/reva#2549](https://github.com/cs3org/reva/pull/2549): Site accounts improvements
  * Enhancement [cs3org/reva#2795](https://github.com/cs3org/reva/pull/2795): Add feature flags "projects" and "share_jail" to spaces capability
  * Enhancement [cs3org/reva#2514](https://github.com/cs3org/reva/pull/2514): Reuse ocs role objects in other drivers
  * Enhancement [cs3org/reva#2781](https://github.com/cs3org/reva/pull/2781): In memory user provider
  * Enhancement [cs3org/reva#2752](https://github.com/cs3org/reva/pull/2752): Refactor the rest user and group provider drivers

   https://github.com/owncloud/ocis/issues/3621
   https://github.com/owncloud/ocis/pull/3552
   https://github.com/owncloud/ocis/pull/3570
   https://github.com/owncloud/ocis/pull/3601
   https://github.com/owncloud/ocis/pull/3602
   https://github.com/owncloud/ocis/pull/3605
   https://github.com/owncloud/ocis/pull/3611
   https://github.com/owncloud/ocis/pull/3637
   https://github.com/owncloud/ocis/pull/3652
   https://github.com/owncloud/ocis/pull/3681

* Enhancement - Update ownCloud Web to v5.5.0-rc.5: [#6854](https://github.com/owncloud/web/pull/6854)

   Tags: web

   We updated ownCloud Web to v5.5.0-rc.5. Please refer to the changelog (linked) for details on
   the web release.

   https://github.com/owncloud/web/pull/6854
   https://github.com/owncloud/ocis/pull/3664
   https://github.com/owncloud/ocis/pull/3680
   https://github.com/owncloud/ocis/pull/3727
   https://github.com/owncloud/ocis/pull/3747
   https://github.com/owncloud/web/releases/tag/v5.5.0-rc.5

**Comment by micbar** [comment]
> > Single File Shares have no download action and thumbnail is broken for the receiver
> 
> What is a single file share? Public link or user? Ticket?

yes, single file user share

The share is displayed in the "Shared" View.

**Comment by micbar** [comment]
@rtest12 The autoprovisioning feature for the ocis IDM was not tackled for this release. Please subscribe to issue #3540. We tackle it in the next beta.
